example/lib/tweet_page.dart

```diff
@@ -55,6 +62,39 @@ class TweetPage extends StatelessWidget {
     );
   }
 
+
+  /// Builds a TweetView from a JSON file
+  Widget buildTweetEmbed(String jsonFile) {
+    return FutureBuilder(
+      future: rootBundle.loadString(jsonFile),
+      builder: (BuildContext context, AsyncSnapshot snapshot) {
+        if (snapshot.hasData) {
+          return Container(
+            margin: EdgeInsets.only(left: 15, right: 15),
+            child: TweetEmbed.fromTweet(
+              
+              Tweet.fromRawJson(
+                snapshot.data,
+              ),
+              backgroundColor: Colors.grey[800],
```

Comment:
you have the dark mode in the embedded section you added to the main screen to the example. Here it would be great to show light versions of tweets and change the background to white (remove line 29 in this file)